.gitignore

```diff
@@ -0,0 +1,17 @@
+# Mac stuff
+.DS_Store
+
+# Compiled Object files
+*.slo
+*.lo
+*.o
+
+# Compiled Dynamic libraries
+*.so
+*.dylib
+
+# Compiled Static libraries
+*.lai
+*.la
+*.a
+
```

Comment:
Why is this .gitignore being added? That is not needed to align the source code, is it? And it is not needed for the Arduino project either, is it?